frontend/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@code-dot-org/frontend",
+  "private": true,
+  "scripts": {
+    "build": "turbo build",
+    "dev": "turbo dev",
+    "lint": "turbo lint",
+    "format": "prettier --write \"**/*.{ts,tsx,md}\""
+  },
+  "devDependencies": {
+    "@storybook/test-runner": "^0.19.1",
+    "prettier": "^3.2.5",
+    "rimraf": "^6.0.1",
+    "turbo": "^2.3.0",
+    "typescript": "5.5.4"
+  },
+  "engines": {
+    "node": ">=22"
+  },
+  "packageManager": "yarn@4.5.1",
+  "workspaces": [
+    "apps/*",
+    "packages/*"
+  ],
+  "version": "0.0.0-alpha.0",
```

Comment:
Should this have a version/etc (this will never be published and is just used to manage the sub-packages, am I right?)

Reply:
You're right, this should not be here.